package.json

```diff
@@ -8,13 +8,13 @@
   ],
   "dependencies": {
     "@edx/brand": "npm:@edx/brand-openedx@1.2.0",
-    "@edx/frontend-component-footer": "11.6.0",
-    "@edx/frontend-enterprise-catalog-search": "3.1.5",
-    "@edx/frontend-enterprise-hotjar": "1.2.0",
-    "@edx/frontend-enterprise-logistration": "2.1.1",
-    "@edx/frontend-enterprise-utils": "2.2.0",
-    "@edx/frontend-platform": "2.6.2",
-    "@edx/paragon": "20.31.1",
+    "@edx/frontend-component-footer": "^12.0.0",
+    "@edx/frontend-enterprise-catalog-search": "^4.0.0",
+    "@edx/frontend-enterprise-hotjar": "^1.3.0",
+    "@edx/frontend-enterprise-logistration": "^3.0.0",
+    "@edx/frontend-enterprise-utils": "^3.0.0",
+    "@edx/frontend-platform": "^4.2.0",
+    "@edx/paragon": "20.28.1",
```

Comment:
Why are we downgrading Paragon?

Reply:
We are not actually, that happened because of conflicts, I have updated.